docs/getting-started.md

```diff
@@ -14,11 +14,11 @@ For information around how to set up React Native, see the [React Native Getting
 Remember to call `react-native init` from the place you want your project directory to live.
 
 ```bat
-npx react-native init <projectName> --template react-native@^0.63.2
+npx react-native init <projectName> --template react-native@^0.64.0
 ```
->To create TypeScript template, run `npx react-native init <projectName> --template react-native-template-typescript@6.5.*`.<br><br>
+>To create TypeScript template, run `npx react-native init <projectName> --template react-native-template-typescript`.<br><br>

```

Comment:
This will use latest TS template, which should be vaguely 1:1 with latest RNW. Should add a version when its available though.